netbox_librenms_plugin/tests/test_librenms_id.py

```
@@ -2,373 +2,1222 @@
     def test_migrate_then_get_returns_value(self):
         from netbox_librenms_plugin.utils import get_librenms_device_id, migrate_legacy_librenms_id
 
-        obj = MagicMock()
-        obj.custom_field_data = {"librenms_id": 55}
-        obj.cf = obj.custom_field_data
+        dev = _dev(55)
+        migrate_legacy_librenms_id(dev, "default")
+        assert get_librenms_device_id(dev, "default") == 55
+
+
+class TestIsLegacyLibreNMSId:
```

Flake8 (7.3.0)

[error] redefinition of unused 'TestIsLegacyLibreNMSId' from line 261

(F811)